lib/Cake/Core/App.php

```diff
@@ -889,6 +889,20 @@ protected static function _packageFormat() {
 	}
 
 /**
+ * Increases the PHP "memory_limit" ini setting by the specified amount
+ * in megabytes  
+ *  
+ * @param string $additionalMb Number in megabytes
+ * @return void
+ */
+	public static function increasePhpMemoryLimit($additionalMb) {
+		$limit = ini_get("memory_limit");
+		if (is_string($limit) && substr($limit, -1) == 'M') {
```

Comment:
That's a good question. I've always just done like 1600M but if PHP accepts
1.6G I can rewrite this.

I mainly did the check just to verify the ini_get was returning what I
expected.
On Sep 6, 2015 1:33 AM, "Christian Winther" <notifications@github.com>
wrote:

> In lib/Cake/Core/App.php
> <https://github.com/cakephp/cakephp/pull/7224#discussion_r38818072>:
>
> > @@ -889,6 +889,20 @@ protected static function _packageFormat() {
> >  	}
> >
> >  /**
> > + * Increases the PHP "memory_limit" ini setting by the specified amount
> > + * in megabytes
> > + *
> > + * @param string $additionalMb Number in megabytes
> > + * @return void
> > + */
> > +	public static function increasePhpMemoryLimit($additionalMb) {
> > +		$limit = ini_get("memory_limit");
> > +		if (is_string($limit) && substr($limit, -1) == 'M') {
>
> so if I allocate 1G of memory, this won't do anything - thats a weird and
> unintended behavior ? :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cakephp/cakephp/pull/7224/files#r38818072>.
>